packages/blockchain-link-utils/src/blockbook.ts

```diff
@@ -87,6 +96,10 @@ export const filterTokenTransfers = (
         });
 };
 
+export const isInstantStaking = ([from, to]: [string, string]) => {
```

Comment:
```suggestion
export const isEthereumStakingInternalTransfer = (from: string, to: string) => {
```
naming and no need for tuple

Reply:
6a6d33b28e105a1f2edf2da7a7e2fe1256ecf019